Implement the traversal subcommands for the H3 cli (#839)
uber/h3: website/docs/api/misc.mdx

Gives the "great circle" or "haversine" distance between pairs of
LatLng points (lat/lng pairs) in radians.

## describeH3Error 

<Tabs
  groupId="language"
  defaultValue="c"
  values={[
    {label: 'C', value: 'c'},
    {label: 'JavaScript (Live)', value: 'javascript'},
  ]
}>
<TabItem value="c">

```c
char *describeH3Error(H3Error err);
```

</TabItem>
<TabItem value="javascript">

:::note

Just read the `.message` property from the caught error, instead.

:::

```js live
function example() {
  let errorMessage = ""
  try {
    h3.cellToChildrenSize("asdf", 9);
  } catch (e) {
    errorMessage = e.message;
  }
  return errorMessage;
}
```

</TabItem>
</Tabs>

> Converts degrees to radians.

Reviewer: Copy/paste issue?

Author: Yep, will fix.

> Provides a human-readable description of an H3Error error code. This function cannot fail, as it just returns a string stating that the H3Error value is itself invalid and allocates zero memory to do so. But because of this it attempting to `free` the returned string will itself cause a failure.

Reviewer 2: ```suggestion
Provides a human-readable description of an H3Error error code. This function cannot fail, as it just returns a string stating that the H3Error value is itself invalid and does not allocate memory to do so. Do not call `free` on the result of this function.
```
I think it might be clearer to explicitly instruct the reader rather than describe what would happen.
